pkg/db/db.go

```diff
@@ -625,7 +625,7 @@ type GetLeasesOutput struct {
 
 // GetLeases takes a set of filtering criteria and scans the Leases table for the matching records.
 func (db *DB) GetLeases(input GetLeasesInput) (GetLeasesOutput, error) {
-	limit := int64(25)
+	var limit int64
```

Comment:
Isn't this setting the default limit to 0? 

According to [docs](https://docs.aws.amazon.com/sdk-for-go/api/service/dynamodb/#ScanInput) without it will stop at 1 megabyte regardless

Reply:
Are you saying that setting the limit to 0 will cause an error, or that we may need to consider adding code in the case that we hit the 1 megabyte limit? I'm also seeing that there is `min:1` do you know if that will cause any issue if we're setting limit to 0?
<img width="1091" alt="Screenshot 2024-02-26 at 9 37 29 AM" src="https://github.com/Optum/dce/assets/55592475/593f1ff1-9d2f-4fd5-be2a-b3d68d1cc377">

Reply:
I don't know exactly how it is handled, a set 0 could be such that it is considered unset, but I would probably expect it to error with an invalid input.

I think what you could do here is ensure Limit is unset in the ScanInput in the case that `if input.Limit > 0 {`, and it will always try to return the max 1 mb of data

Reply:
@bjfish25 I made a change for the limit. Please check again